src/components/TimelineItem.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <section class="timeline-item">
     <div class="item">
-      <span :style="checkForColourDots(itemTimeline.color)" class="dot" />
+      <span :style="getBackgroundColour(item.color)" class="dot" />
```

Comment:
be careful, you use `item`, but data is `itemTimeline`. Change this and then we can merge it